.github/workflows/release.yaml

```diff
@@ -0,0 +1,73 @@
+name: release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: "1.15"
+
+      - uses: actions/cache@v2
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: ${{ runner.os }}-go-
+
+      - uses: actions/cache@v2
+        with:
+          path: ./testbin
+          key: ${{ runner.os }}-testbin
+
+      - uses: actions/cache@v2
+        with:
+          path: ./bin
+          key: ${{ runner.os }}-bin
+
+      - name: Build image
+        run: make docker-build
+
+      - name: Export release name
+        run: |
+          echo "NEW_RELEASE=$(make get-new-release)" >> $GITHUB_ENV
+
+      - name: Login to quay.io/3scale
+        if: ${{ env.NEW_RELEASE != '' }}
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.REGISTRY_USER }}
+          password: ${{ secrets.REGISTRY_PASSWORD }}
+
+      - name: Push new operator image
+        if: ${{ env.NEW_RELEASE != '' }}
+        run: make docker-push
+
+      - name: Login to quay.io/3scaleops
+        if: ${{ env.NEW_RELEASE != '' }}
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.CATALOG_REGISTRY_USER }}
+          password: ${{ secrets.CATALOG_REGISTRY_PASSWORD }}
```

Comment:
This is a bit weird because we have the operator image in the 3scale quay namespace but the bundle and catalog images are in the 3scaleops namespace. Maybe it's a good moment to move them to the same namespace.

Reply:
Agree with moving all images to the same namespaces now that there is no problem on having different catalogs on the same quay namespace.

I think it is better to use `3scale` namespace, it is possible that it is a powerful account, so possibly less limited that free account `3scaleops`

Reply:
https://github.com/3scale/saas-operator/issues/86 to track this task.